fs/littlefs/Kconfig

```
@@ -4,3 +4,13 @@ config FS_LITTLEFS
 	depends on !DISABLE_MOUNTPOINT
 	---help---
 		Build the LITTLEFS file system. https://github.com/ARMmbed/littlefs.
```

```suggestion
		Build the LITTLEFS file system. https://github.com/littlefs-project/littlefs.
```
Please, could you also update the project URL?